build/common.m4

```diff
@@ -75,6 +75,7 @@ AC_DEFUN([GP_CHECK_MINGW],
 		*mingw*)
 			AC_DEFINE([WIN32], [1], [we are cross compiling for WIN32])
 			AM_CONDITIONAL([MINGW], true)
+			AC_SUBST([docdir], '${prefix}/doc/geany-plugins')
```

Comment:
Oops, this was just a left-over in this PR. I actually built the provided test binaries without this.
Thanks for noticing it and sorry. Fixed in https://github.com/eht16/geany-plugins/commit/953d478f2cbb1b06d730e421b6673bdfdd4564dd.